### ISSUE TITLE
Locking terminal

### ISSUE THREAD

**Issue by ndrluis** [twe4ked/rspec-nc#22]
When the spec suite has been finished the terminal are locking and I need to force quit to unlock my terminal.

```
OS: macOS Sierra
Terminal: Iterm2
Shell: ZSH
Ruby version: 2.2.3
Rspec Version: 3.4.0
```

**Comment by ndrluis** [comment]
https://github.com/julienXX/terminal-notifier/issues/112

**Comment by twe4ked** [comment]
I can reproduce it here. I'll take a look, thanks for reporting.

**Comment by twe4ked** [comment]
Looks like it's related to this: https://github.com/julienXX/terminal-notifier/issues/172

**Comment by twe4ked** [comment]
Looks like the bug has been fixed in version 1.7.1 of terminal-notifier. Re-open if you're still having issues.
